client/packages/system/src/Manage/Preferences/EditPage/EditPreferencesPage.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import {
+  LocaleKey,
+  useBreadcrumbs,
+  useParams,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { LineEditBase } from './LineEditBase';
+import { useAvailablePreferences } from '../api';
+import { EditPreference } from './EditPreference';
+
+export const EditPreferencesPage = () => {
+  const t = useTranslation();
+  const { key } = useParams();
+  const { setCustomBreadcrumbs } = useBreadcrumbs();
+
+  const { data } = useAvailablePreferences();
+
+  useEffect(() => {
+    if (key)
+      setCustomBreadcrumbs({
+        1: t(`preference.${key}` as LocaleKey),
+      });
+  }, [key]);
+
+  const selectedPref = data?.find(d => key === d.key);
+
+  return (
+    <LineEditBase currentKey={key ?? ''} prefs={data ?? []}>
```

Comment:
I honestly think this should be a normal table, where we can see all preferences, and then we go to matrix view for individual preferences for stores etc..